modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/actions/Actions.tsx

```diff
@@ -212,9 +212,11 @@ const Actions = ({dataSet, namespace, spritemap}: IDataSetSectionProps) => {
 					onClick: ({processClose}: {processClose: Function}) => {
 						processClose();
 
-						fetch(item.actions.delete.href, {
+						const url = `${API_URL.ACTIONS}/${item.id}`;
```

Comment:
Setup fixed actions url, in the same way it is done in Filters and Sorting.
In the future, we should apply the `filterItemActions` utility we have in FDS to display only available actions for Filters, Sorting and Actions.